app/e2e/tests/dashboard/test.list.ts

```diff
@@ -0,0 +1,53 @@
+// https://playwright.dev/docs/test-parallel#use-a-test-list-file
+import { test } from '@local/common/utils/fixtures';
+import { PlaywrightOrganizationsPage } from '@local/common/pages';
+import organizerTests from './organizer-dashboard.spec';
+import userTests from './user-dashboard.spec';
+
+// Tests in ilst can be run in parallel as long as the setup/teardown is the same for all test suites.
+// If the setup/teardown is different for any test suite, then tests should be run in serial.
+// https://playwright.dev/docs/test-parallel#serial-mode
+// Use `test.describe.configure({ mode: 'serial' });` to run tests in serial.
+
+// Dashboard tests setup and teardown
+// Create organization and events that can be used to test the dashboard
+test.beforeAll(async ({ browser, device }) => {
+    const context = await browser.newContext();
+    const page = await context.newPage();
+    const organizationsPage = await new PlaywrightOrganizationsPage(page, device).create(browser);
+
+    // Create organization
+    await organizationsPage.goto();
+    await organizationsPage.clickOnCreateOrganization();
+    await organizationsPage.fillInOrganizationName('Test Organization');
+    await organizationsPage.submitOrganizationForm();
+
+    // Create ongoing event
+    const today = new Date();
```

Comment:
Might be better to set the date in the page class via the constructor so it is set when the page is created and can be used in all necessary methods directly rather than passing it in here for today and tomorrow.